server/services/calculateReleaseDatesService.ts

```diff
@@ -283,6 +283,16 @@ export default class CalculateReleaseDatesService {
         )}`
       )
     }
+    if (rules.includes('ERSED_MAX_PERIOD')) {
+      const ruleSpecificAdjustment = rulesWithExtraAdjustments.ERSED_MAX_PERIOD
+      return CalculateReleaseDatesService.createAdjustmentRow(
+        releaseDate,
+        ReleaseDateType.ERSED,
+        `${longDateFormat(unadjustedDate)} ${daysArithmeticToWords(adjustedDays)} ${arithmeticToWords(
+          ruleSpecificAdjustment
+        )}`
+      )
+    }
```

Comment:
Picky - it's only the rule specific adjustment value that changes here, could be made a method to reduce the duplication

Reply:
Yes - that's picky. I thought about it but went for expedience over cleanliness.